tap_google_analytics/client.py

```diff
@@ -255,12 +326,8 @@ def _make_request(self, method, url, params=None, data=None):
             response = self.session.post(url, headers=headers, params=params, json=data, timeout=self.request_timeout)
         else:
             response = self.session.request(method, url, headers=headers, params=params, timeout=self.request_timeout)
-
-        error_message = _is_json(response) and response.json().get("error", {}).get("message")
```

Comment:
The removed code was throwing exceptions based on status codes and on the should_retry() method which checks if the status code is retryable, if yes raises the exception through raise_for_status(). However, we have now moved to custom exceptions, which will be raised based on corresponding status codes.